### PR TITLE
Add Stripe memo to DC purchase modal 

### PR DESCRIPTION
**Pull request by ambuznego**
image
image

With memo:
image

With no memo:
image

### CODE REVIEW

File: assets/js/actions/dataCredits.js [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
-)
+import * as rest from "../util/rest";
+
+export const createCustomerIdAndCharge = (amountUSD, description) => () =>
+  rest
+    .post("/api/data_credits/create_customer_and_charge", {
+      amountUSD: convertToString(amountUSD),
+      description,
+    })
+    .then(({ data }) => data);
```
Comment (ambuznego) [inline assets/js/actions/dataCredits.js:9]: adding `description` here

File: assets/js/actions/dataCredits.js [hunk 2]
```diff
@@ -1,61 +1,74 @@
...
+
+export const createCharge = (amountUSD, description) => () =>
+  rest
+    .post("/api/data_credits/create_charge", {
+      amountUSD: convertToString(amountUSD),
+      description,
+    })
+    .then(({ data }) => data);
```
Comment (ambuznego) [inline assets/js/actions/dataCredits.js:17]: adding `description` here

File: assets/js/actions/dataCredits.js [hunk 3]
```diff
@@ -1,61 +1,74 @@
...
+  (cost, cardType, last4, paymentId, description) => () =>
+    rest
+      .post("/api/data_credits/create_dc_purchase", {
+        cost,
+        cardType,
+        last4,
+        paymentId,
+        description,
+      })
+      .then(() => {});
```
Comment (ambuznego) [inline assets/js/actions/dataCredits.js:48]: adding `description` here.... rest is linting

File: assets/js/components/billing/PurchaseCreditModal.jsx [hunk 4]
```diff
@@ -403,6 +419,34 @@ class PurchaseCreditModal extends Component {
...
+              <div
+                style={{
+                  display: "flex",
+                  flexDirection: "row",
+                  alignItems: "center",
+                }}
+              >
+                Memo
+                <Popover
+                  content="For Stripe payments only, this will show on the emailed receipt."
```
Comment (vicmgs) [inline assets/js/components/billing/PurchaseCreditModal.jsx:439]: Should we mention something about VAT?

**Review by vicmgs — APPROVED** [review]

File: assets/js/components/billing/PurchaseCreditModal.jsx (hunk 4 above)
Comment (ambuznego) [reply assets/js/components/billing/PurchaseCreditModal.jsx:439]: good point, i wanted to leave it broad but i can add VAT as an example of what they could put there

File: assets/js/components/common/PacketGraph.jsx [hunk 5]
```diff
@@ -66,28 +66,28 @@ class PacketGraph extends Component {
...
+            display: true,
           },
           min: -120,
           max: 20,
         },
         x: {
           title: {
-            text: 'Time Past in Seconds',
-            display: true
+            text: "Time Passed in Seconds",
```
Comment (ambuznego) [inline assets/js/components/common/PacketGraph.jsx:77]: main change in this file, rest is linting.

Comment (vicmgs) [reply assets/js/components/common/PacketGraph.jsx:77]: lgtm

**Pull request merged**